cloudsmith_cli/core/api/metrics.py

```diff
@@ -14,6 +14,29 @@ def get_metrics_api():
     return get_api_client(cloudsmith_api.MetricsApi)
 
 
+def organization_entitlement_usage_metrics(owner=None, **kwargs):
+    """List token usage metrics."""
+    client = get_metrics_api()
+
+    api_kwargs = {}
+    api_kwargs.update(
```

Comment:
NAB: Could just use comprehension below as initialiser rather than declare a new dict and update?

Reply:
I agree it would be cleaner however this is the pattern that's been used across multiple commands. If we were to change it, it should be updated everywhere for consistency but there is probably a bigger refactoring pass that is needed for the CLI that this could be part of.